src/superannotate/lib/core/usecases/images.py

```diff
@@ -849,6 +849,7 @@ def execute(self) -> Response:
 
 class UploadImagesToProject(BaseInteractiveUseCase):
     MAX_WORKERS = 10
+    LIST_NAME_CHUNK_SIZE = 500
```

Comment:
it's useless

Reply:
It's  not useless, I assign it to a variable called CHUNK_SIZE in line 1027 in the `filter_paths` function, because writing `UploadImagesToProject.LIST_NAME_CHUNK_SIZE` is very very inconvenient.

Reply:
I saw, how I called it as a class variable of a different class. The fact that it worked was a coincidence. see fixed.